runtime/caml/memory.h

```diff
@@ -18,7 +18,7 @@
 #ifndef CAML_MEMORY_H
 #define CAML_MEMORY_H
 
-#ifndef CAML_NAME_SPACE
+#ifndef CAML_INTERNALS
```

Comment:
Is this correct?

Reply:
I just hit it in practice. This change is a bit annoying.

Reply:
Please can we start to switch these threads to fresh issues or (for one-liners) possible PRs

Reply:
@dra27 Maybe @silene's comment did not really add to the discussion, but it is important to keep the cost low of getting this patch feedback, and being able to have a discussion on the diff is very useful. It should not be discouraged. Opening separate issues or PRs on lots of tiny details is a real time sink. For my own review comments I will be waiting for some feedback here before proceeding.

Reply:
I felt @silene was adding to the discussion - but fresh conversation threads in this PR are going to run a high risk of being dropped, because they're hard to track.